ipywidgets/static/widgets/js/layout.js

```diff
@@ -36,18 +36,15 @@ define([
                 'align_items',
                 'align_self',
                 'bottom',
+                'border',
```

Comment:
Besides, I like the idea of being able to group items together in a widget dashboard using a border, like this.

<img width="635" alt="sliders" src="https://cloud.githubusercontent.com/assets/2397974/12128999/e5bb7692-b3ce-11e5-9648-47001a5a633a.png">
)